### PR TITLE
fix: fix update bug of filters (non-inherit case)  & fix bug for non-inherit case in `getInitialWidgetInheritOptions`

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [x] Not that difficult
- [ ] Approved feature branch merge to master

### CODE REVIEW

File: apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardWidgetOptionDropdown.vue [hunk 1]
```diff
@@ -212,15 +212,36 @@ const initSelectedMenuItems = async (): Promise<SelectDropdownMenuItem[]> => {
...
+                    [dataName]: selected[0].name,
+                };
+            } else {
+                widgetOptions[propertyName] = selected[0].name;
+            }
+        } else if (propertyName.includes('filters')) {
+            widgetOptions.filters = {
+                ...widgetOptions.filters,
+                [dataName]: selected.map((item) => item.name),
+            };
```
Comment (yuda110) [inline apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardWidgetOptionDropdown.vue:232]: description: 

after
```
{
  filters: {
    provider: ['aws', 'google_cloud']
  }
}
```

before
```
{ filters.provider: ['aws', 'google_cloud'] }
```

Comment (WANZARGEN) [reply apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardWidgetOptionDropdown.vue:232]: Thanx for your description

File: apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardWidgetOptionDropdown.vue [hunk 2]
```diff
@@ -212,15 +212,36 @@ const initSelectedMenuItems = async (): Promise<SelectDropdownMenuItem[]> => {
...
         } else {
             widgetOptions[propertyName] = selected.map((item) => item.name);
         }
+    }
+
+    // delete case
+    if (propertyName.includes('filters')) {
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardWidgetOptionDropdown.vue:239]: suggestion: I suggest below: 

```suggestion
    if (propertyName.startsWith('filters.')) {
```

**Review by WANZARGEN — APPROVED** [review]
LGTM!

**Pull request merged**